docs/section19/19-10.md

```diff
@@ -0,0 +1,84 @@
+                - Case 2: Elif the building has refrigerated cases, the building segment building area type is retail and no economizer is modeled in the baseline or proposed and so the system may meet exception #2 outcome is UNDETERMINED (this is exception #2): `elif (hvac.fan_system.AirEconomizer == Null OR hvac.fan_system.AirEconomizer.type == "FIXED_FRACTION") AND hvac in hvac_system_exception_2_list: outcome = "UNDETERMINED" and raise_message "Fail unless any of the zones served by the baseline system <insert hvac.id> in the proposed design include supermarket open refrigerated case-work systems that will be affected by using outdoor air for cooling (G3.1.2.6 exception #2).:"`                  
+                - Case 3: Elif the system primarily serves computer rooms and the baseline system type is 3 or 4 and an economizer is modeled then fail (this is exception #3): `elif (hvac.fan_system.AirEconomizer != Null AND hvac.fan_system.AirEconomizer.type != "FIXED_FRACTION") AND hvac_sys_type_b in ["SYS-3", "SYS-3a", "SYS-3b", "SYS-3c", "SYS-4"] AND hvac.id in HVAC_systems_primarily_serving_comp_rooms_list: outcome = "FAIL" and raise_message "This system [hvac.id] appears to meet the criteria associated with Section G3.1.2.6 exception #3 which is that an economizer shall not be modeled in the baseline for systems that serve computer rooms complying with Section G3.1.2.6.1. "`  
+                - Case 4: Elif the building has refrigerated cases, the building segment building area type is retail and an economizer is modeled in the baseline but not the proposed and so the system likely meets exception #2 so outcome is UNDETERMINED (this is exception #2): `elif (hvac.fan_system.AirEconomizer != Null AND hvac.fan_system.AirEconomizer.type != "FIXED_FRACTION") AND hvac in hvac_system_exception_2_list: outcome = "UNDETERMINED" and raise_message "This system [hvac.id] appears to meet the criteria associated with Section G3.1.2.6 exception #2 which is that an economizer shall not be modeled in the baseline for systems where the use of outdoor air for cooling will affect supermarket open refrigerated case-work systems and the proposed system does not include an economizer. An economizer has been modeled in the baseline when it appears this exception may apply. Manual check recommended."`                     
+                - Case 5: Elif there is an AirEconomizer object associated with the fan and the AirEconomizer type is no fixed fraction (which indicates to economizer) then pass: `elif hvac.fan_system.AirEconomizer != Null AND hvac.fan_system.AirEconomizer.type != "FIXED_FRACTION": outcome = "PASS"`  
```

Comment:
I think there is a few typos and the text should read "Elif there is an AirEconomizer object associated with the fan and the AirEconomizer type is no**T** fixed fraction (which indicates THAT THERE IS AN economizer) then pass

Reply:
Ha! It must have been late when I wrote that!